src/redaction/redactor.ts

```
@@ -11,116 +11,117 @@ import { defaultRedactionConfig, RedactionController } from './config';
-        // Reload environment configuration to pick up any changes
-        const envConfig = RedactionController.getEnvironmentConfig();
-        this.config = { 
-            ...defaultRedactionConfig, 
-            ...envConfig,
-            ...newConfig 
-        };
-    }
+  static updateConfig(newConfig: Partial<RedactionConfig>): void {
+    // Reload environment configuration to pick up any changes
+    const envConfig = RedactionController.getEnvironmentConfig();
+    this.config = {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)